src/main/java/app/bpartners/api/model/mapper/PaymentRequestMapper.java

```diff
@@ -15,10 +15,17 @@
 import app.bpartners.api.repository.fintecture.model.FPaymentRedirection;
 import app.bpartners.api.repository.jpa.model.HPaymentRequest;
 import java.time.Instant;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
 
 @Component
 public class PaymentRequestMapper {
+  private String dashboardUrl;
```

Comment:
No. 

Do not put it inside a Mapper. But inside a general conf more specific. Create a new conf (as DashboardConf) if necessary.